drivers/sensor/bma280/bma280_trigger.c

```diff
@@ -220,8 +232,8 @@ int bma280_init_interrupt(struct device *dev)
 
 	gpio_pin_configure(drv_data->gpio,
 			   DT_INST_0_BOSCH_BMA280_INT1_GPIOS_PIN,
-			   GPIO_DIR_IN | GPIO_INT | GPIO_INT_LEVEL |
-			   GPIO_INT_ACTIVE_HIGH | GPIO_INT_DEBOUNCE);
+			   DT_INST_0_BOSCH_BMA280_INT1_GPIOS_FLAGS
+			   | GPIO_INT_LEVEL_ACTIVE);
```

Comment:
I guess this was supposed to be `GPIO_INT_DEBOUNCE`.

Reply:
Every time I leave `GPIO_INT_DEBOUNCE` in place @mnkp tells me to remove it.  In this case I did.  Since AFAICT there is no provision for disabling this sensor in a way that might cause the signal to float I think it's unnecessary.

Reply:
My point was rather that `GPIO_INT_LEVEL_ACTIVE` does not make sense here, as this is a call to `gpio_pin_configure`, not `gpio_pin_interrupt_configure`. That's why I thought it was supposed to be `GPIO_INT_DEBOUNCE` instead. `GPIO_INPUT` also seems to be missing.

Reply:
That's more clear.  The fragment shown didn't include the function, and it is indeed `GPIO_INPUT` that was intended.  Thanks; fixed.